src/main/resources/application.conf

```diff
@@ -38,113 +46,224 @@ etc-client {
-    pooling-services-timeout = 5.seconds
+
+    ommer-pool-query-timeout = 5.seconds
   }
 
   blockchain {
+    # Frontier block number
     frontier-block-number = "0"
+
+    # Homestead fork block number
+    # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2.md
     homestead-block-number = "1150000"
+
+    # EIP-150 fork block number
+    # Doc: https://github.com/ethereum/EIPs/issues/150
     eip150-block-number = "2500000"
+
+    # EIP-160 fork block number
+    # Doc: https://github.com/ethereum/EIPs/issues/160
     eip160-block-number = "3000000"
+
+    # Difficulty bomb pause block number
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
     difficulty-bomb-pause-block-number = "3000000"
+
+    # Difficulty bomb continuation block number
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
     difficulty-bomb-continue-block-number = "5000000"
 
-    // Doc: https://blog.ethereum.org/2016/07/20/hard-fork-completed/
+    # DAO fork block number (Ethereum HF/Classic split)
+    # https://blog.ethereum.org/2016/07/20/hard-fork-completed/
     dao-fork-block-number = "1920000"
-    dao-fork-block-total-difficulty = "39490964433395682584"
+
+    # The hash of the accepted DAO fork block
     dao-fork-block-hash = "94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f"
 
+    # Starting nonce a an empty account. Some networks (like Morden) use different values.
     account-start-nonce = "0"
 
+    # The ID of the accepted chain
     chain-id = "3d"
 
+    # Custom genesis JSON file path
+    # null value indicates using default genesis definition that matches the main network
     custom-genesis-file = null
 
-    # ECIP-1017
+    # Monetary policy parameters
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1017.md
     monetary-policy {
+      # Block reward in the first era
       first-era-block-reward = "5000000000000000000"
+
+      # Monetary policy era duration in number of blocks
       era-duration = 5000000
+
+      # Rate at which rewards get reduced in successive eras.
+      # Value in range [0.0, 1.0]
       reward-reduction-rate = 0.2
     }
   }
 
-  fast-sync {
+  sync {
+    # Whether to enable fast-sync
     do-fast-sync = true
+
+    # Interval for updating peers during sync
     peers-scan-interval = 3.seconds
+
+    # Duration for blacklisting a peer. Blacklisting reason include: invalid response from peer, response time-out, etc.
     blacklist-duration = 30.seconds
+
+    # Retry interval when not having enough peers to start fast-sync
     start-retry-interval = 5.seconds
+
+    # Retry interval for resuming fast sync after all connections to peers were lost
     sync-retry-interval = 5.seconds
+
+    # Response time-out from peer during sync. If a peer fails to respond within this limit, it will be blacklisted
     peer-response-timeout = 10.seconds
```

Comment:
I didn't have such a need, though I admit it seems small. But other than `etc-client` -> `grothendieck` I tried to avoid changing any values in this PR.